nbconvert/filters/tests/test_markdown.py

```diff
@@ -175,8 +183,8 @@ def test_markdown2rst(self):
         tokens[1] = r'\*\*test'
 
         for index, test in enumerate(self.tests):
-            self._try_markdown(markdown2rst, test, tokens[index])
-
+            self._try_markdown(partial(convert_pandoc, from_format='markdown',
```

Comment:
compare this to https://github.com/jupyter/nbconvert/pull/436/files#diff-fa9719444368d3ed717eaea36d19e9a7R33 and https://github.com/jupyter/nbconvert/pull/436/files#diff-72a549c7ebe4e2a80383f8351ef80d4fR65

In this case from_format is being treated as keyword argument, but in the other cases, it's being implicitly passed as a positional argument (but in one case in the first position and in the other case the second position).

     partial(convert_pandoc, from_format='markdown',to_format='rst')

vs. 

     convert_pandoc('markdown', 'latex')

vs. 

    convert_pandoc(s, 'markdown_strict', 'latex')